### PR TITLE
Add configuration files for Ponicode tests

### PR DESCRIPTION
**Pull request by sguillia**
Hi!

This pull request adds configuration files for [Ponicode](ponicode.com/).

Here are instructions to get started
https://drive.google.com/file/d/1T6lOF-fxHn860pjmDFe398z3l3bulqRr/view?usp=sharing

Best,
The Ponicode team

### CODE REVIEW

File: src/tsconfig.json [hunk 1]
```diff
@@ -3,6 +3,8 @@
     "compilerOptions": {
         "types": [],
         "baseUrl": "./",
-        "rootDir": "./"
+        "rootDir": "./",
+        "outDir": "../.temp/manualBuildCore",
```
Comment (deltakosh) [inline src/tsconfig.json:7]: @sebavan : can this be a problem? (To define an outdir?)

Comment (deltakosh) [reply src/tsconfig.json:7]: @sguillia Did you try to compile the project with `npm run build` to ensure your changes are not breaking the build?

Comment (sebavan) [reply src/tsconfig.json:7]: yup all good :-)

**Comment by sebavan** [comment]
at worst @sguillia force it in your gulp task

**Pull request merged**